invenio.cfg

```diff
@@ -200,6 +200,10 @@ CELERY_BEAT_SCHEDULE = {
             "files_query": "invenio_app_rdm.utils.files.checksum_verification_files_query",
         },
     },
+    "update-collections-size": {
+        "task": "invenio_rdm_records.collections.tasks.update_collections_size",
+        "schedule": timedelta(hours=1),  # Every hour
```

Comment:
To be discussed:

- the frequency. For now, it's a quick task so hourly seems OK
- the queue where to run the task (e.g. low vs main queue)